social/twitter/27-twitter.js

```diff
@@ -398,8 +398,8 @@ module.exports = function(RED) {
                 if (msg.hasOwnProperty("payload")) {
                     node.status({fill:"blue",shape:"dot",text:"twitter.status.tweeting"});
 
-                    if (msg.payload.length > 140) {
-                        msg.payload = msg.payload.slice(0,139);
+                    if (msg.payload.length > 280) {
```

Comment:
You'd be better off using twitter-text 2.0 (when it is released) to validate the length rather than assuming a hard-coded 280 limit. This also will not account for CJK characters which have a different weighting under the new counting algorithm https://developer.twitter.com/en/docs/developer-utilities/twitter-text

Reply:
Yeah... I see this as the quick fix ahead of a bigger rework of the node (#361)